code/modules/mob/living/carbon/xenomorph/strains/castes/drone/healer.dm

```diff
@@ -61,6 +61,10 @@
 
 /datum/action/xeno_action/activable/apply_salve/use_ability(atom/target_atom)
 	var/mob/living/carbon/xenomorph/xeno = owner
+
+	if(xeno.m_intent == MOVE_INTENT_STALK)
+		return FALSE
```

Comment:
What is special about this activatable ability and no other that you need to check this in its use_ability?

Reply:
pretty sure if i put it after it will affect CD even if the ability is not used

Reply:
removed those lines since the restrictions will be inside "can_use"